### PR TITLE
Configure GH Actions for running tests before merging into development branch

### PR DESCRIPTION
**Pull request by KeremZaman**
- Github workflow was created for running tests on each push and merge into development branch

**NOTE**: Please do not merge this request before PR #231 is merged because this branch is created from eventposts-search.

### CODE REVIEW

**Review by berkaydoner — APPROVED** [review]
As far as I know, this configuration should work. I can also help you connect the frontend unit tests similarly, when we have some time.

File: backend/Dockerfile [hunk 1]
```diff
@@ -7,6 +7,9 @@ RUN pip install -r requirements.txt
 ADD . /
 WORKDIR /
 
-CMD python manage.py makemigrations authentication && python manage.py makemigrations eventposts && python manage.py migrate && python manage.py runserver 0.0.0.0:9000
+## POSTGRES WAIT SCRIPT FROM https://github.com/ufoscout/docker-compose-wait
+ADD https://github.com/ufoscout/docker-compose-wait/releases/download/2.9.0/wait /wait
+RUN chmod +x /wait
+CMD /wait && python manage.py makemigrations authentication && python manage.py makemigrations eventposts && python manage.py migrate && python manage.py test eventposts && python manage.py runserver 0.0.0.0:9000
```
Comment (supiket) [inline backend/Dockerfile:13]: I am assuming we will remove "eventposts" from the test command here.

File: backend/eventposts/tests.py [hunk 2]
```diff
@@ -1,3 +1,169 @@
...
+
+        self.duration_between_45_100_games = self.football_games
+        self.games_created_by_user = EventSerializer(EventPost.objects.all(), many=True).data
+        self.creation_date_today = EventSerializer(EventPost.objects.all(), many=True).data
+        self.num_players_between_1_5_games = self.football_games
+        self.num_spectators_between_3_5_games = EventSerializer(EventPost.objects.filter(title="basketball event"),
+                                                          many=True).data
+
+        self.skill_between_2_4 = self.football_games
+
```
Comment (supiket) [inline backend/eventposts/tests.py:81]: The test coverage looks great, I see pretty exhaustive tests here.

**Review by supiket — APPROVED** [review]
Well-written tests. Contribution to Github Actions looks great. And test database configurations in this PR is contributing to tests for comments & answers. Thank you for your effort.

File: backend/Dockerfile (hunk 1 above)
Comment (KeremZaman) [reply backend/Dockerfile:13]: Since it will be fixed by your branch, I'm leaving that line as it is for this PR. Thanks for reviewing!

**Pull request merged**